src/main/java/com/club_board/club_board_server/config/jwt/TokenExceptionHandlerFilter.java

```diff
@@ -34,15 +34,18 @@ private void handleBusinessException(HttpServletRequest request, HttpServletResp
         response.setContentType(MediaType.APPLICATION_JSON_VALUE);
         response.setCharacterEncoding("UTF-8"); // HttpServletResponse: ISO-8859-1 인코딩 사용하기 때문에 한글 출력을 위해 UTF-8 설정
 
-        // CORS 관련 헤더를 명시적으로 추가
-        response.setHeader("Access-Control-Allow-Origin", "*");
+        String origin = request.getHeader("Origin");
+        if(origin == null || origin.isEmpty()){
+            origin = "https://chipsatbooks.vercel.app";
+        }
```

Comment:
```suggestion
        String origin = request.getHeader("Origin");
        
        List<String> allowedOrigins = List.of("https://chipsatbooks.vercel.app", "http://localhost:5173", "http://localhost:5174", "http://localhost:3000");
        
        if (origin == null || !allowedOrigins.contains(origin)) {
            origin = "https://chipsatbooks.vercel.app";
        }
```
요청 온 `origin`을 그대로 담아 보내주는 건 모든 도메인을 허용하는 cors 설정이 되는 것 같습니다. 물론 기존 코드에서는 와일드카드로 모두 허용해주었긴 했지만, 변경하는 김에 이런 식의 코드는 어떨까요?